pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/UserGovernanceDao.java

```diff
@@ -112,6 +112,19 @@ default int deleteAllGovernancesForProxy(long proxyUserId) {
         return findGovernancesByParticipantAndStudyGuids(participantGuid, studyGuid).filter(Governance::isActive);
     }
 
+    @UseStringTemplateSqlLocator
+    @SqlQuery("queryGovernancesByParticipantAndStudyIds")
+    @RegisterConstructorMapper(Governance.class)
+    @RegisterConstructorMapper(GrantedStudy.class)
+    @UseRowReducer(GovernanceWithStudiesReducer.class)
```

Comment:
Some really nice jdbi work here--filling out `grantedStudies` inside `Governance` on-the-fly.  Along with the `select_all_governances_with_granted_studies` bits in the `stg` file, this is really slick.